Sources/CXExtensions/WeakAssign.swift

```diff
@@ -0,0 +1,90 @@
+import CXShim
+import CXUtility
+
+extension Publisher where Failure == Never {
+    
+    public func assign<Root: AnyObject>(to keyPath: ReferenceWritableKeyPath<Root, Output>, weaklyOn object: Root) -> AnyCancellable {
+        let assign = Subscribers.Assign(object: object, keyPath: keyPath)
```

Comment:
`WeakAssign`?

Reply:
Can't find a way to withdraw the approval...

Reply:
Fixed